src/server.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+
+const PORT = process.env.PORT || 8080;
+
+const server = http.createServer((req, res) => {
+  const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+  const pathName = normalizedUrl.pathname.slice(1) || 'index.html';
```

Comment:
Why to set `index.html` here? Will this path satisfy condition of `if` block on 12 line?